checklists/silver.md

```diff
@@ -1,49 +1,35 @@
 ## Silver Certification
 
-**BOLD text is different from Bronze**
+- [ ] **All items from Bronze**
 
 > TODO: Insert example Silver cookie-cutter here.
 
 ### Documentation
 
-- [ ] **Background/significance of program.**
-- [ ] Description of what program does is listed.
-- [ ] **A tutorial that showcases the program's usage. If multiple usages, examples of those are shown also.**
-- [ ] Description of **all** input parameters.
-- [ ] Description of outputs.
-    - Advanced usage
-    - Document functions intended to be used by users.
+- [ ] Background/significance of program
+- [ ] A tutorial that showcases the program's typical intended usage
+- [ ] Advanced usage is described
+- [ ] Thorough description of required and optional input parameters
+- [ ] Document public functions (docstring / help coverage ≥ 20%)
```

Comment:
How are "public" functions different from "user-facing" or "intended to be used by user"?

Reply:
Public functions _are_ user-facing functions intended to be used by users (as opposed to private functions intended to be used by functions). The difference for that one between bronze and silver is the coverage threshold. We should probably use consistent language across these :sweat_smile: